package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "reactjs-mentoring",
+  "version": "1.0.0",
+  "description": "ReactJS Global Mentoring Program 2020Q3",
+  "private": true,
+  "main": "index.jsx",
+  "dependencies": {
+    "cross-env": "^7.0.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-redux": "^7.2.1",
+    "react-router": "^5.2.0",
+    "redux": "^4.0.5",
+    "regenerator-runtime": "^0.13.7"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.0",
+    "@babel/core": "^7.1.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@testing-library/react": "^10.4.7",
+    "babel-jest": "^26.2.2",
+    "babel-loader": "^8.0.2",
+    "case-sensitive-paths-webpack-plugin": "^2.3.0",
+    "css-loader": "^1.0.0",
+    "file-loader": "^6.0.0",
+    "html-webpack-plugin": "^4.3.0",
+    "jest": "^26.2.2",
+    "mini-css-extract-plugin": "^0.9.0",
+    "react-hot-loader": "^4.12.21",
+    "react-test-renderer": "^16.13.1",
+    "style-loader": "^0.23.0",
+    "terser-webpack-plugin": "^3.0.8",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-merge": "^5.1.0"
+  },
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server",
+    "dev": "cross-env NODE_ENV=development webpack",
```

Comment:
Just a suggestion(not necessary to implement)
I noticed that if you have multiple configurations for different environments it is much easier to use smth like this:
`build:dev`
`build:prod`

As a result you need to type smth like this in terminal window
`npm run build:dev`

Reply:
Good note, I agree.